pyoxidizer/src/py_packaging/distribution.rs

```diff
@@ -2,29 +2,46 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+use copy_dir::copy_dir;
 use itertools::Itertools;
 use serde::{Deserialize, Serialize};
 use sha2::{Digest, Sha256};
-use slog::warn;
-use std::collections::BTreeMap;
+use slog::{info, warn};
+use std::collections::{BTreeMap, HashMap};
+use std::env;
 use std::fs;
 use std::fs::{create_dir_all, File};
-use std::io::{Cursor, Read};
+use std::io::{BufRead, BufReader, Cursor, Read};
 use std::path::{Path, PathBuf};
 use url::Url;
+use which::which;
 
+use super::distutils::prepare_hacked_distutils;
 use super::fsscan::{
     find_python_resources, is_package_from_path, walk_tree_files, PythonFileResource,
 };
 use super::resource::{PythonResource, ResourceData, SourceModule};
+
 use crate::licensing::NON_GPL_LICENSES;
 
+#[cfg(windows)]
+const PYTHON_EXE_BASENAME: &str = "python.exe";
+
+#[cfg(unix)]
+const PYTHON_EXE_BASENAME: &str = "python3";
+
 #[cfg(windows)]
 const PIP_EXE_BASENAME: &str = "pip3.exe";
 
 #[cfg(unix)]
 const PIP_EXE_BASENAME: &str = "pip3";
 
+#[cfg(windows)]
+const VENV_MODULE: &str = "virtualenv";
```

Comment:
support for virtualenv should be removed as there isnt any case which needs it now.